support-payment-api/src/main/scala/backend/AmazonPayBackend.scala

```diff
@@ -65,7 +65,7 @@ class AmazonPayBackend(
   private def handleResponse(response: Either[AmazonPayApiError, AuthorizationDetails] , amazonPayRequest: AmazonPayRequest, clientBrowserInfo: ClientBrowserInfo)  = {
     response.toEitherT[Future]
       .leftMap { error =>
-        logger.error(s"Something went wrong with ${amazonPayRequest.paymentData.orderReferenceId}: ${error.getMessage}")
+        logger.info(s"Something went wrong with AmazonPay orderReferenceId: ${amazonPayRequest.paymentData.orderReferenceId}")
         cloudWatchService.recordFailedPayment(error, PaymentProvider.AmazonPay)
```

Comment:
Do we lose anything here by not having the `orderReferenceId` in the error?

Reply:
we will still have it in the preceding `info` log, but we're leaving it to `recordFailedPayment` to decide whether it should be logged as an `error`. I think it should be easy enough for anyone investigating an alarm to find this